cernopendata/modules/fixtures/data/records/cms-condition-data-Run2012B-C.json

```diff
@@ -0,0 +1,116 @@
+[
+{
+  "abstract": {
+    "description": "<p>Condition database includes non-event-related\n      information (Alignment, Calibration, Temperature, etc) and parameters for\n      simulation/reconstruction/analysis software.</p>\n      <p> These files are needed when running a CMSSW job on the Open Data VM. They are automatically read from\n      the /cvmfs/cms-opendata-conddb.cern.ch runtime, you do not need to download these files from here separately.</p>\n      <p>FT53_V21A_AN6 is to be used with the collision data of RunB 2012 and FT53_V21A_AN6_RUNC with the collision data of RunC 2012 (with the event range of the open data, the file for the full event range for 2012 are also stored\n      under FIXME on /cvmfs/cms-opendata-conddb.cern.ch)</p> \n      <p>For 2012 CMS Simulated Datasets see:</p>", 
+    "links": [
+      {
+        "recid": "1804"
+      }
+    ]
+  }, 
+  "accelerator": "CERN-LHC", 
+  "collaboration": {
+    "recid": "451", 
+    "name": "CMS collaboration"
+  },
+  "collections": [
+    "CMS-Condition-Data"
+  ], 
+  "date_created": "2012", 
+  "date_published": "2017", 
+  "experiment": "CMS", 
+  "files": [
+    {
+      "checksum": "FIXME", 
+      "uri": "root://eospublic.cern.ch//eos/opendata/cms/Run2012B/db/FT53_V21A_AN6.db", 
```

Comment:
See https://github.com/cernopendata/opendata.cern.ch/issues/1263#issuecomment-348399790 regarding size and checksum.

However, we don't have `*.tgz` files yet; perhaps you can keep only `*.db` for now, and we merge that way...